tide_core.install

```diff
@@ -1548,3 +1548,29 @@ function tide_core_update_8064() {
     }
   }
 }
+
+/**
+ * Add contact field to user account(business name, phone, notes).
+ */
+function tide_core_update_8065() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_core') . '/config/install'];
+  _tide_import_single_config('field.storage.user.field_business_contact_number', $config_location);
+  _tide_import_single_config('field.storage.user.field_business_name', $config_location);
+  _tide_import_single_config('field.storage.user.field_notes', $config_location);
+  _tide_import_single_config('field.field.user.user.field_business_contact_number', $config_location);
+  _tide_import_single_config('field.field.user.user.field_business_name', $config_location);
+  _tide_import_single_config('field.field.user.user.field_notes', $config_location);
+}
+
+/**
+ * Update user form display and authenticated user permission.
+ */
+function tide_core_update_8066() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_core') . '/config/install'];
+  _tide_import_single_config('user.role.authenticated', $config_location);
+  _tide_import_single_config('user.role.site_admin', $config_location);
+  _tide_import_single_config('core.entity_form_display.user.user.default', $config_location);
+}
+
```

Comment:
please use `\Drupal::entityTypeManager()` to create the storage and field for both of the update hook. there are other examples in this file that you can follow. e.g - https://github.com/dpc-sdp/tide_core/blob/1082c243f94fce61cc242110595db4735169e3f5/tide_core.install#L1397